src/tools/cephfs/cephfs-shell

```diff
@@ -781,8 +782,9 @@ exists.')
     rmdir_parser.add_argument('paths', help='Directory Path.', nargs='+')
     rmdir_parser.add_argument('-p', '--parent', action='store_true',
                               help='Remove parent directories as necessary. \
-When this option is specified, no error is reported if a directory has any \
-sub-directories, files')
+                              When this option is specified, no error is \
+                              reported if a directory has any sub-directories,\
+                              files')
```

Comment:
So, is the error just suppressed or contained files are removed too? Perhaps the explanation needs to be more clear. Also, `subdirectory, files` should be `subdirectory or files`. And if we remove all kinds of files, which I think the code does, just replacing `subdirectory, files` by `files` should be good.

Reply:
On second thought, why is `rmdir` attempting to delete files? It should not delete files, right?. GNU rmdir's man page says "remove empty directories" but our help text just says "Remove Directory." -

```
CephFS:~/>>> help rmdir
usage: rmdir [-h] [-p] paths [paths ...]

Remove Directory.

positional arguments:
  paths         Directory Path.

optional arguments:
  -h, --help    show this help message and exit
  -p, --parent  Remove parent directories as necessary. When this option is
                specified, no error is reported if a directory has any sub-
                directories, files
```

I think not deleting files would be better.

Reply:
cephfs-shell rmdir does not delete files. It only deletes empty directories.
But there is some issue with -p option.

Reply:
I will fix the -p option and help message together in a new PR. As this one is just concerned with 'rmdir *'.

Reply:
Ok.